concepts/05 UI Components/HtmlEditor/10 Formats/33 Customize Built-In Formats and Modules/20 Extend.md

```diff
@@ -136,3 +136,5 @@ In the following code, the `strike` format is extended so that the stricken out
     export default App;
 
```

Comment:
Please update the following line above:

Line 3:
Please change "stricken out text" to "strikethrough text".